src/Settings/components/fontFace.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import styles from "../styles";
+import { setFontFace, FONT_FACES } from "../../common/actions";
+import { useSelector } from "react-redux";
```

Comment:
As mentioned above too, move third party imports before the internal imports

Reply:
fixed